erpnext/manufacturing/doctype/production_plan/production_plan.json

```diff
@@ -35,6 +35,7 @@
   "for_warehouse",
   "download_materials_required",
   "get_items_for_mr",
+  "include_sub_assembly_with_exploded_item",
```

Comment:
When this checkbox is checked, shouldn't the checkbox  for all the rows of child table be checked ?

Reply:
This checked box if checked and includes exploded item check in the child table will fetch subassembly. if this checked box is checked and  all row material don't have any include exploded item check it will work with normal behavior fetch only BOM ITem